src/features/MentorPage/MentorChips/MentorChips.tsx

```diff
@@ -0,0 +1,74 @@
+import { ChipProps } from '../../../components/Chip/types';
+import styled from 'styled-components';
+import Chip from '../../../components/Chip';
+import ShowMoreChips from './ShowMoreChips';
+import React from 'react';
+
+/**
+ * Logic to show skill chips. Array of skill names is
+ * mapped and passed to Chip one by on, where styles and layout
+ * will be added.
+ */
+
+const MentorSkillChips = ({ items }: { items: Array<ChipProps> }) => {
+  const [shouldShowAllSkills, setShowAllSkills] = React.useState(false);
+
+  const handleShowMoreSkillsChange = () =>
+    setShowAllSkills(!shouldShowAllSkills);
+
+  //change the first letter of every skill to uppercase, then alphabetize
+  const upperCaseSkills = items.map(item => {
+    const newItem: ChipProps = {
+      text: item.text.charAt(0).toUpperCase() + item.text.slice(1),
+    };
+    return newItem;
+  });
+  const sortedSkills = [...upperCaseSkills].sort((a, b) =>
+    a.text.localeCompare(b.text),
+  );
```

Comment:
Instead of uppercasing the first letter like this, you can just uppercase/lowercase the item when you sort.

const sortedSkills = [...upperCaseSkills].sort((a, b) =>
    a.text.toLowerCase().localeCompare(b.text.toLowerCase()),
  );
  
 No need for variable upperCaseSkills

Reply:
We could also create an utility/helper function for sorting array of strings, which could have a few unit tests. =P

Reply:
I'm still going to leave this as it is, since it would look weird if some of the words are written with uppercase first letter and some not.